src/lib/email-service.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable class-methods-use-this */
+const nodemailer = require("nodemailer")
+
+module.exports = class Email {
+	/** Represents the email sender
+     * @constructor
+     * @param {object} user - The user object 
+     */
+	constructor(user) {
+		this.to = user.email
+		this.firstName = user.firstName
+		this.from = process.env.EMAIL_FROM
+	}
+
+	/**
+     * Selectively creates a new transport based on the environment
+     * @returns a new nodemailer transport
+     */
+	newTransport() {
+
+		if (process.env.NODE_ENV === 'development') {
+			return nodemailer.createTransport({
+				host: process.env.EMAIL_HOST,
```

Comment:
let's use the `config` function to access the envs so we at least get an error if the env is not set

Reply:
https://github.com/tbf-challenge/teamwork-be/blob/main/src/config.js

Reply:
done @jocrah